universal-application-tool-0.0.1/app/controllers/dev/AwsStorageDevControllerStrategy.java

```diff
@@ -0,0 +1,35 @@
+package controllers.dev;
```

Comment:
like should this controller class be in /dev/ is it only used for dev stuff?

Reply:
This class is only used for the AWS (localhost is the AWS emulator) dev file upload view

Reply:
oo i think im just confused like don't we use it https://github.com/seattle-uat/civiform/pull/1752/files#diff-dd152546f04a2023052ca2e60280bd5fad6875511262887f7c8e1259b35a8458R59 to specify that we are using aws/azure apis

Reply:
Specifying whether to use azure or AWS happens in the application.conf file. CloudStorageModule is for dependency injection. We're checking the conf file and then binding the appropriate dev storage view/controller class. We're not doing that for any prod code because we only currently only support AWS in prod, we haven't gotten around to the Azure implementations yet.

Reply:
Does that make sense? Am I answering your question or not really?

Reply:
Similarly to the interface name, maybe rename this to `AwsLocalstackFileStorageStrategy`?

Reply:
Done, and changed the Azure one to ```AzuriteFileStorageStrategy``` does that sound okay?